_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/pages/_searchapp.scss

```diff
@@ -0,0 +1,270 @@
+rhdp-search-app {
+
+  .hide {
+    display: none;
+  }
+
+  .show {
+    display: block;
+  }
+
+  .mobile {
+    display: none;
+  }
+
+  .loading {
+    background: url("/images/icons/ajax-loader.gif") center 80px no-repeat;
+    min-height: 250px;
+  }
+}
+
+.rhd-c-search-page-header {
+  margin-bottom: 1rem;
+
+  .pf-c-input-group {
+    .pf-c-button.pf-m-danger {
+      &:after {
+        //remove pf after causing blue underline on hover
+        display: none;
+      }
+    }
+  }
+}
+
+rhdp-search-result-count {
+  display: block;
+  margin-bottom: 1em;
+  font-weight: 600;
+  font-size: 1.2em;
+}
+
+rhdp-search-sort-page {
+  display: block;
+  margin-bottom: .5rem;
+  clear: both;
+}
+
+rhdp-search-filters {
+  margin-bottom: 1rem;
+  display: block;
+
+  // Active filter labels (not checkboxes)
+  .activeFilters {
+    rhdp-search-filter-item[active] {
+      display: inline-block;
+
+      .pf-c-label {
+        cursor: pointer;
+        margin: .25rem;
+      }
+    }
+  }
+
+  .title {
+    background-color: var(--pf-global--palette--black-100, #fafafa);
+    color: var(--rhd-theme--color-text, #151515);
+    font-family: RedHatDisplay;
+    font-size: 16px;
+    font-weight: 700;
+    padding: 1rem;
+  }
+
+  .control {
+    border: 1px solid var(--rhd-theme--color--ui-border, #ededed);
+  }
+
+  &[type="modal"] {
+    display: none;
+  }
+
+  // Mobile styles
+  @media only screen and (max-width: $pf-global--breakpoint--md) {
+
+    // Prevent rhdp-search-filters from showing below the footer.
+    flex: none;
+    border: none;
+
+    .mobile {
+      display: block;
+    }
+
+    .hide-on-mobile,
+    .desktop {
+      display: none;
+    }
+
+    .control {
+      display: flex;
+      flex-direction: column;
+      width: 100%;
+      height: 100%;
+      padding-top: 51px;
+      background: rgba(0, 0, 0, 0.5);
+      border: none;
+      z-index: 99;
+      right: 100%;
+      position: absolute;
+      top: 100px;
+    }
+
+    &[type="modal"] {
+      display: block;
+
+      .modal {
+        transition: 0.5s ease-in-out;
+        transform: translateX(100%);
+      }
+
+      .cover {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        height: 100%;
+        padding-top: 100px;
+        background: rgba(0, 0, 0, 0.5);
+        border: none;
+        z-index: 999;
```

Comment:
why such a large number for `z-index`?

Reply:
This PR wasn't a complete refactor of the search app; It was only to remove Foundation and repair/update what was necessary.  We will be looking into PFE-Search and Fusion eventually, and a full css/js code review can happen then?  Please let us know otherwise if this causes any issues.